components/SettingsControls/src/SettingsExpander/SettingsExpander.xaml

```diff
@@ -93,7 +92,7 @@
                                                     IsClickEnabled="False" />
                             </muxc:Expander.Header>
                             <muxc:Expander.Content>
-                                <Grid>
+                                <Grid CornerRadius="{TemplateBinding CornerRadius}">
```

Comment:
When setting the `CornerRadius` to `24`, this now happens:
![image](https://github.com/CommunityToolkit/Windows/assets/9866362/0c222d24-84f7-48bb-83c3-a5370908b998)

Reply:
@0x5bfa would it make sense to add a `Converter` here that translates the `CornerRadius`? E.g. a `CornerRadius` of `24` would be converted to `24,0,0,24`?

Reply:
Yes definitely, I'd like to make it more generally so everyone can use everywhere. Thus, using ConverterParameter like 'ConverterParameter=1001' true and false combination - it means 24 0 0 24 according to the input.